src/plugins/data/common/search/aggs/agg_config.ts

```diff
@@ -21,12 +21,13 @@ import _ from 'lodash';
 import { i18n } from '@kbn/i18n';
 import { Assign, Ensure } from '@kbn/utility-types';
 
-import { FetchOptions, ISearchSource } from 'src/plugins/data/public';
+import { ISearchSource } from 'src/plugins/data/public';
```

Comment:
@lukeelmers `ISearchSource` needs to be moved to `common` as well, so we can avoid circular deps

Reply:
👍  My next PR for search source (after the one removing the `elasticsearch-browser` dependency for msearch) will be to move everything to `common`, including the types.

Reply:
Could we still merge this one though @lukeelmers ?

Reply:
Yeah I don't see why not